pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -343,7 +342,7 @@ func (l *listUsersQuery) expandDirect(
 
 	filteredIter := storage.NewFilteredTupleKeyIterator(
 		storage.NewTupleKeyIteratorFromTupleIterator(iter),
-		validation.FilterInvalidTuples(typesys), // why filter invalid here?
+		validation.FilterInvalidTuples(typesys), // ignore tuples that don't conform to the model
```

Comment:
```suggestion
		validation.FilterInvalidTuples(typesys),
```

Comment isn't really helping, let's just remove.